tools/fix_auth/fix_auth.rb

```diff
@@ -75,6 +75,10 @@ def fix_database_yml
       FixDatabaseYml.run({:hardcode => options[:password]}.merge(run_options))
     end
 
+    def load_rails
+      require File.expand_path("../../../config/application.rb", __FILE__)
```

Comment:
Don't you want to run the initializers?  Typically, tools in the tools directory require config/environment to load rails, which is the same as above ^ but it also calls `Vmdb::Application.initialize!`

Reply:
I'll move this comment to the master PR